pilot/pkg/bootstrap/options.go

```diff
@@ -78,6 +78,7 @@ type PilotArgs struct {
 	KeepaliveOptions         *istiokeepalive.Options
 	// ForceStop is set as true when used for testing to make the server stop quickly
 	ForceStop bool
+	BasePort  int
```

Comment:
what is this for?

Reply:
VMs and local debugging. With istiod ( and pilot ) you can now start multiple pilots/istiod in debugger or on a local machine. Quite convenient for debug, and I think for running pilot on VM
it's also nice to be able to pack multiple variants. 

Testing too.